src/main/java/offeneBibel/osisExporter/ObOsisGeneratorVisitor.java

```diff
@@ -137,17 +137,33 @@ else if(node.getNodeType() == ObAstNode.NodeType.quote) {
             if(m_quoteCounter>0)
             {
                 m_quoteCounter++;
-                m_currentFassung.append("»<q level=\"" + m_quoteCounter + "\" marker=\"\"" + end);
+                
+                m_currentFassung.append("»");
```

Comment:
Shouldn't that go after the if? I guess the » shouldn't be part of the highlighting.